py-polars/polars/lazyframe/frame.py

```diff
@@ -1236,6 +1249,8 @@ def bottom_k(
             per column by passing a sequence of booleans.
         nulls_last
             Place null values last.
+        maintain_order
+            Whether the order should be maintained if elements are equal.
```

Comment:
Can you add a note that this is more expensive and blocks streaming.

Reply:
done